### ISSUE TITLE
How to contribute

### ISSUE THREAD

**Issue by GuillaumeMilan** [edgurgel/httpoison#474]
At least I could not find it. But I think it could be great if the support team could provide a workflow to follow when we want to contribute to the repo.

- Shall we open an issue before creating a PR?
- Must we create some tests if we add new functions

**Comment by edgurgel** [comment]
Yeah good point! I will add some info on contributing. But to answer the question: Opening issues is fine and creating PRs without an issue is also fine. And ideally adding tests 👍
